formbricks/formbricks
# Feat/delete user account

## As a user I want to delete my account



Fixes #349



## Type of change




## How should this be tested?



- Login to the platform
- Click on the Settings option on the navigation bar 
- On the side navigation, click on profile below the account menu. 
- Click on the delete my account button 
- Confirm your deletion 

## Checklist
